composer.json

```diff
@@ -3,5 +3,10 @@
     "description": "Offer to your clients the best interest in e-Commerce by using InfinitePay as your payment method.",
     "require": {
         "php": ">=5.6"
+    },
+    "autoload": {
+        "psr-4": {
+            "Woocommerce\\InfinitePay\\": "src/"
+        }
```

Comment:
This is applied only for development environment or Wordpress also uses what is described here somehow?

Reply:
It's need to the plugin working with namespace as the new version does

Reply:
for any environment.